ES/ESmy022.xml

```diff
@@ -382,22 +330,87 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                                                                        text area and margin dimensions taken from
+                                                                        <locus target="#4r"/>. 9-10 characters per written line.</note>
+                                                                     <dimensions unit="mm" xml:id="textarea">
+                                                                        <height>108</height>
+                                                                        <width>98</width>
+                                                                        <dim type="intercolumn"/>
+                                                                     </dimensions>
+                                                                     <dimensions unit="mm" xml:id="margin">
+                                                                        <dim type="top">10</dim>
+                                                                        <dim type="bottom">30</dim>
+                                                                        <dim type="right">15</dim>
+                                                                        <dim type="left">10</dim>
+                                                                        <dim type="intercolumn"/>
+                                                                     </dimensions>
+                                                              <ab type="pricking">Pricking and ruling are visible.</ab>
+                                                                <ab type="ruling" subtype="pattern">1A-1A/0-0/0-0/A</ab>
+                                                                <ab type="pricking">Primary pricks are visible.</ab>
+                                                                <ab type="pricking">Ruling pricks are visible.</ab>
+                                                                <ab type="pricking">Structural pricks are not visible.</ab>
+                                                                <ab type="ruling">The upper line is written above the ruling.</ab>
+                                                                <ab type="ruling">The bottom line is written above the ruling.</ab>
+                                                              </layout>
+                                                                  </layoutDesc>
+                                                                  </objectDesc>
+
+                                                                            <handDesc>
+                                                                           <handNote xml:id="h2" script="Ethiopic">
+                                                                           <locus from="3r" to="6r"/>
+                                                                            <seg type="script">Late twentieth or twenty first-century script.</seg>
```

Comment:
no `<date>` element here?

Reply:
I do not understand exactly what is the function of this element. There is no way to date it in any other way as said under "script".

Reply:
Oh, I see. I did, no problem. In this place I did not tag the date before, and we did not tag it here in the catalogue of Dayr as-Suryan